README.md

```diff
@@ -2,42 +2,339 @@
+    lx       = ly           # domain length in x
+    ni       = nx, ny       # number of cells
+    li       = lx, ly       # domain length in x- and y-
+    di       = @. li / ni   # grid step in x- and -y
+    origin   = 0.0, 0.0     # origin coordinates
+    xci, xvi = lazy_grid(di, li, ni; origin=origin) # nodes at the center and vertices of the cells
+    dt       = Inf 
+```
+Initialisation of the Rheology utilising [GeoParams.jl](https://github.com/JuliaGeodynamics/GeoParams.jl). The Rheology can be taylored to the specific problem with different creep laws and material parameters (see [GeoParams.jl](https://github.com/JuliaGeodynamics/GeoParams.jl)) or the miniapps in the [convection folder](miniapps/convection).
+
+```julia
+    # Physical properties using GeoParams ----------------
+    τ_y     = 1.6           # yield stress. If do_DP=true, τ_y stand for the cohesion: c*cos(ϕ)
+    ϕ       = 30            # friction angle
+    C       = τ_y           # Cohesion
+    η0      = 1.0           # viscosity
+    G0      = 1.0           # elastic shear modulus
+    Gi      = G0/(6.0-4.0)  # elastic shear modulus perturbation
+    εbg     = 1.0           # background strain-rate
+    η_reg   = 8e-3          # regularisation "viscosity"
+    dt      = η0/G0/4.0     # assumes Maxwell time of 4
+    el_bg   = ConstantElasticity(; G=G0, Kb=4)
+    el_inc  = ConstantElasticity(; G=Gi, Kb=4)
+    visc    = LinearViscous(; η=η0) 
+    pl      = DruckerPrager_regularised(;  # non-regularized plasticity
+        C    = C,
+        ϕ    = ϕ, 
+        η_vp = η_reg,
+        Ψ    = 0,
+    ) 
+    rheology = (
+        # Low density phase
+        SetMaterialParams(;
+            Phase             = 1,
+            Density           = ConstantDensity(; ρ = 0.0),
+            Gravity           = ConstantGravity(; g = 0.0),
+            CompositeRheology = CompositeRheology((visc, el_bg, pl)),
+            Elasticity        = el_bg,
+
+        ),
+        # High density phase
+        SetMaterialParams(;
+            Density           = ConstantDensity(; ρ = 0.0),
+            Gravity           = ConstantGravity(; g = 0.0),
+            CompositeRheology = CompositeRheology((visc, el_inc, pl)),
+            Elasticity        = el_inc,
+        ),
+    )
+```
+Initialisation of the Stokes arrays and the necesseary allocations. The rheology is computed with `compute_viscosity!` which is a function from `JustRelax.jl` and computes the viscosity according to the strain rate and the phase ratios.
+
+```julia
+    # Initialize phase ratios -------------------------------
+    radius       = 0.1
+    phase_ratios = PhaseRatio(ni, length(rheology))
+    init_phases!(phase_ratios, xci, radius)
+
+    # STOKES ---------------------------------------------
+    # Allocate arrays needed for every Stokes problem
+    stokes    = StokesArrays(ni, ViscoElastic)
+    pt_stokes = PTStokesCoeffs(li, di; ϵ=1e-6,  CFL = 0.75 / √2.1)
```

Comment:
It would be good here state that the PT coefficients come from [Ludo's et al GMD paper](https://gmd.copernicus.org/articles/15/5757/2022/)